homeassistant/components/azure_devops/__init__.py

```diff
@@ -83,12 +83,12 @@ class AzureDevOpsDeviceEntity(AzureDevOpsEntity):
     @property
     def device_info(self) -> DeviceInfo:
         """Return device information about this Azure DevOps instance."""
+        devid = f"{self.organization}.{self.project}"
         return {
             "identifiers": {
                 (
                     DOMAIN,
-                    self.organization,
-                    self.project,
+                    devid,
```

Comment:
Changing it now means that a new device will be created as it won't be able to identify the existing device.

Reply:
That makes sense, I have changed it so that mypy are pleased, and existing devices are identified. Mypy have its advantages, but sometimes "type: ignore" is the better choice.